discovery/gossiper_test.go

```diff
@@ -3571,15 +3571,76 @@ func TestSplitAnnouncementsCorrectSubBatches(t *testing.T) {
 		announcementBatch := make([]msgWithSenders, batchSize)
 
 		splitAnnouncementBatch := splitAnnouncementBatches(
-			subBatchSize,
-			announcementBatch)
+			subBatchSize, announcementBatch,
```

Comment:
Nit: looks like the `TestSplitAnnouncementCorrectSubBatches` changes are a fixup of https://github.com/lightningnetwork/lnd/pull/2985/commits/f4a9edca7e107e9af31ad8b44c8ed9c5f78c2792?